metadata/configuration/concepts/1286_referral_form.csv

```diff
@@ -1,17 +1,19 @@
 Uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_order:1286
 c5b20e93-56c8-45e5-b65b-2b42ee49ecb0,,,Date de référence,Referral Date,Date de référence,Referral Date,Misc,Date,
+a2f31381-448b-4ca4-9639-9df83e740442,,,Consultation externe,Outpatient,Consultation externe,Outpatient,Misc,N/A,
 1b7bc29f-188f-4103-8bac-90deb019e1ea,,30dcc74a-5554-482f-a5a0-9f6fb3c2a556;a2f31381-448b-4ca4-9639-9df83e740442,Nom du service,Name of service,Nom du service,Name of service,Misc,Coded,
```

Comment:
Did we forget to get the French equivalent of `30dcc74a-5554-482f-a5a0-9f6fb3c2a556` concept? I see outpatient is done. Please confirm